### PR TITLE
Run all of the timm models shards in the periodic

### PR DESCRIPTION
**Pull request by ezyang**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #92743

Signed-off-by: Edward Z. Yang <ezyang@meta.com>

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

### CODE REVIEW

File: .ci/pytorch/test.sh [hunk 1]
```diff
@@ -343,13 +343,13 @@ test_inductor_benchmark_perf() {
 
 # No sharding for the periodic job, we don't care if latency is bad
 test_aot_eager_all() {
-  PYTHONPATH=$(pwd)/torchbench test_aot_eager_benchmark torchbench 0
-  test_aot_eager_benchmark huggingface 0
-  test_aot_eager_benchmark timm_models 0
+  PYTHONPATH=$(pwd)/torchbench test_aot_eager_benchmark torchbench ""
+  test_aot_eager_benchmark huggingface ""
+  test_aot_eager_benchmark timm_models ""
```
Comment (ezyang) [inline .ci/pytorch/test.sh:348]: We do an `-n` test on the shard ID, which is true for "" but false for 0 lol

**Review by kit1980 — APPROVED** [review]

**Pull request closed without merging**